File: 3.EDD/7°Version/Java/Financiera.java
import java.util.Scanner;
public class Financiera {
    public static void main(String[] args){
        Scanner mex = new Scanner(System.in);
        System.out.println("Arbol binario JAVA");
        ArbolBinario coleccion = new ArbolBinario();
        int respuesta;
        do {
            Prestamo P1 = new Prestamo();
            System.out.println("\n");
            System.out.println("Datos del acreedor");
            System.out.println("\n");
            System.out.println("Ingresa los nombres: ");
            P1.setNombre(mex.nextLine());
            System.out.println("Ingresa los apellidos: ");
            P1.setApellido(mex.nextLine());
            System.out.println("Ingresa un numero telefonico: ");
            P1.setTelefono(mex.nextLine());
            System.out.println("Ingresa un correo electronico (Opcional): ");
            P1.setCorreo(mex.nextLine());
            System.out.println("Ingrese la cantidad a prestar: ");
            P1.setPrestamo(mex.nextDouble());
            System.out.println("Ingresar el porcentaje de interes: ");
            P1.setPorcentaje(mex.nextDouble());
            System.out.println("Ingresa la cantidad de a pagar: ");
            P1.setSemana(mex.nextDouble());
            coleccion.agregar(P1);
            System.out.println("Para continuar oprima 1, para finalizar 0: ");
            respuesta = mex.nextInt();
            mex.nextLine();
        } while (respuesta != 0);
        System.out.println("\n[PRE]_________________________________________P R E O R D E N_________________________________________[PRE]\n");
        coleccion.preorden();
        System.out.println("\n[IN]___________________________________________I N O R D E N__________________________________________[IN]\n");
        coleccion.inorden();
        System.out.println("\n[POST]_______________________________________P O S T O R D E N_______________________________________[POST]\n");
        coleccion.postorden();
        mex.close();
    }
    
}

File: 3.EDD/7°Version/C++/Financiera.cpp
#include <iostream>
#include "ArbolBinario.h"
using namespace std;

Prestamo capturaPrestamo(){

    string palabras;
    double cantidad;
    Prestamo P1;
    // Datos del acreedor-------------------
    cout << "Ingresar datos del acreedor";
    cout << "\n \n";
    cout << "Ingresar nombres: ";
    getline(cin,palabras);
    P1.setNombre(palabras);
    cout << "Ingresar apellidos: ";
    getline(cin,palabras);
    P1.setApellido(palabras);

    //Datos del pretamo-------------------------
    cout <<"\n \n";
    cout << "Ingresar la cantidad a prestar: ";
    cin >> cantidad;
    P1.setPrestamo(cantidad);
    cout << "Ingresar el porcentaje de interes (%): ";
    cin >> cantidad;
    P1.setPorcentaje(cantidad);
    cout << "Ingresar la cantidad de semanas a pagar: ";
    cin >> cantidad;
    P1.setSemana(cantidad);
    return P1;
}

int main(){
    ArbolBinario Prestamos;
    cout << "Arbol binario";
    int respuesta;
    do{
        cout << "\n\n----------------------------------------------[CAPTURA]\n";
        Prestamo nuevoprestamo;
        nuevoprestamo= capturaPrestamo();
        Prestamos.agregar(nuevoprestamo);
        cout << "Oprime 1 continuar, 0 para finalizar= ";
        cin >> respuesta;
        cin.ignore();
    } while (respuesta !=0);

    cout << "\n\n-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=[PRE.ORDEN]-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=\n";
    Prestamos.preorden();
    cout << "\n\n-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=[IN.ORDEN]-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-\n";
    Prestamos.inorden();
    cout << "\n\n-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=[POS.ORDEN]-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=\n";
    Prestamos.posorden();
    return 0;
    
}